app/src/main/java/com/mealsmadeeasy/ui/home/suggestions/SuggestionsAdapter.kt

```diff
@@ -6,12 +6,17 @@ import android.view.View
 import android.view.ViewGroup
 import android.widget.ImageView
 import android.widget.TextView
-import android.widget.Toast
 import com.mealsmadeeasy.R
 import com.mealsmadeeasy.model.Meal
 import com.squareup.picasso.Picasso
 
-class SuggestionsAdapter(meals: List<Meal> = emptyList()) : RecyclerView.Adapter<SuggestionViewHolder>() {
+private lateinit var suggestionListener : SuggestionClickListener
```

Comment:
This should be inside the class. Otherwise it becomes static which is super bad. I'd think you can implement this by making it `val suggestionClickListener` in the constructor unless you really need to reassign it.